research/denoised_smoothing/certification_output/sigma_0.25_denoiser_mse

```diff
@@ -0,0 +1,501 @@
+idx	label	predict	radius	correct	time
```

Comment:
This file (and other files under `certification_output`) are the Colab notebook's output, right? If so, I think we don't need to check in them.

Reply:
Yes. I have kept the outputs for safe housekeeping.